test/data/marriage-abroad-files.yml

```diff
@@ -123,14 +125,25 @@ lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in
+lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_macao.govspeak.erb: ba677dd8eaea321c0f47d8e3fa72089b
+lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_morocco.govspeak.erb: 26bd88af00f90e3971f9fb4e9ca37692
 lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_north_korea.govspeak.erb: 66649a60a573f59f856eff3066f567d1
+lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_norway.govspeak.erb: 32aa56f9e970759f7b852208ba55100e
 lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_oman.govspeak.erb: 8f9519a97e516f2a3eb36cc816b43dcc
 lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_other_countries.govspeak.erb: 49fa07fccc73a085c20c4695841bc306
+lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_peru.govspeak.erb: 2148db48081e8b91421ff9f39efa1e60
+lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_philippines.govspeak.erb: ea7fac85596c2b09fc6c9aad7c501bc0
 ? lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_poland_when_residing_in_uk_or_poland.govspeak.erb
 : 5b832527c2503714db0a361056e8f61c
+lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_qatar.govspeak.erb: 6c8368aa17f6a92665ae3c3e697fd672
 lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_saudi_arabia.govspeak.erb: 31bedffa052920e5bbdc67951a842401
 ? lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_slovenia_when_residing_in_uk_or_slovenia.govspeak.erb
 : e4ff64b9911bd6b8d2d980f6311db678
+lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_south_korea.govspeak.erb: 9e8bce8e41660249e19361fae264462b
+lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_thailand.govspeak.erb: 35b669954974ce4ae10b92d0ac3a8479
+lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_turkey.govspeak.erb: 30f42fd8b80b4a25fc464c6e5b4f44d9
+lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_united_arab_emirates.govspeak.erb: a959cbe9a2171aeb49ff2a786f54a9bd
+lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_vietnam.govspeak.erb: 7de688fb25bf157202b76e1dc960a07c
 lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_yemen.govspeak.erb: 75931823b0856f97977823a467d25ec9
 lib/smart_answer_flows/marriage-abroad/outcomes/outcome_same_sex_civil_partnership.govspeak.erb: 32a91cace111cb531a1e895307785f71
 lib/smart_answer_flows/marriage-abroad/outcomes/outcome_same_sex_civil_partnership_in_affirmation_countries.govspeak.erb: 959fc1ca1916dbf62a8f2e1e8aed0d39
```

Comment:
When I spoke to you earlier, you said you'd run the regression tests for all countries before and after the changes in this PR. If that's the case, I think you should remove the "TODO" in the commit note and explain what you've done.